e2e/tests/flag_test.go

```diff
@@ -117,12 +117,19 @@ func TestFlagMITMDomains(t *testing.T) {
 	})
 }
 
-func TestFlagDenyDomain(t *testing.T) {
-	newClient(t, "https://www.google.com").GET("/").ExpectStatus(http.StatusForbidden)
-	newClient(t, httpbin).GET("/status/200").ExpectStatus(http.StatusOK)
+func TestFlagDenyDomains(t *testing.T) {
+	t.Run("include(google)", func(t *testing.T) {
+		newClient(t, "https://www.google.com").GET("/").
+			ExpectStatus(http.StatusForbidden)
+	})
+
+	t.Run("exclude (httpbin)", func(t *testing.T) {
+		newClient(t, httpbin).GET("/status/200").
+			ExpectStatus(http.StatusOK)
+	})
```

Comment:
This change does not resonate with the commit name. 
Also make subtests names consistent - spaces.

Reply:
Spaces makes it look like `foo_(` vs `foo(`

Reply:
Updated the commit msg